jnigen/lib/src/tools/android_sdk_tools.dart

```diff
@@ -24,15 +45,8 @@ class AndroidSdkTools {
     return null;
   }
 
-  static Future<String?> getAndroidSourcesPath(
```

Comment:
This used to point to `sources` package installed from `sdkmanager`. But unlike android stubs sources, which are sometimes complete, these are never complete. So just removed this functionality.